openrr-planner/src/planner/joint_path_planner.rs

```diff
@@ -210,13 +236,20 @@ where
         let num_smoothing = self.num_smoothing;
         rrt::smooth_path(
             &mut path,
-            |angles: &[N]| self.is_feasible_with_self(using_joints, angles),
+            |angles: &[N]| self.is_feasible_with_self(&using_joints, angles),
             step_length,
             num_smoothing,
         );
         Ok(path)
     }
 
+    /// Synchronize joint positions of the planning robot model with the reference robot
+    pub fn sync_joint_positions(&self) {
```

Comment:
It might be better to use a more descriptional name, because it's difficult to understand what `sync` means, especially about the direction. (plan to ref? ref to plan?).
So, I suggest `sync_joint_positions_from_reference`, but if you have a better name, please tell me!

Reply:
Thank you, and I agree with your comment on this.
I suggest `sync_joint_positions_with_reference` instead, considering the use of `synchronize` (e.g. synchronize A with B).

Reply:
Nice!